### ISSUE TITLE
generate bearer token failed

### ISSUE THREAD

**Issue by zxyyy0119** [karmada-io/dashboard#38]
I installed Karmada by Karmada command-line tool on my own cluster (karmadactl init) and joined cluster with name member 2 to Karmada.
![image](https://github.com/karmada-io/dashboard/assets/142280847/c679a9ee-0294-4571-9fc0-fd7364c898b8)

Then I tried to install Karmada Dashboard by following the steps on github and I found out that I only had karmada-apiserver.config file (seems like I am missing something about karmada-host). Also, when I generated bearer token, it showed this error. 
![image](https://github.com/karmada-io/dashboard/assets/142280847/035bc0ac-94b3-4408-85f2-567a4af5cea5)

**Comment by RainbowMango** [comment]
The Karmada dashboard project is still far from being available, and the development of this project has actually stagnated due to the lack of front-end developers. We are also seeking developers who are willing to contribute to this project.

**Comment by haiker2011** [comment]
same problems

**Comment by warjiang** [comment]
> I installed Karmada by Karmada command-line tool on my own cluster (karmadactl init) and joined cluster with name member 2 to Karmada. ![image](https://private-user-images.githubusercontent.com/142280847/276524806-c679a9ee-0294-4571-9fc0-fd7364c898b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMzkyMzcsIm5iZiI6MTcxMDEzODkzNywicGF0aCI6Ii8xNDIyODA4NDcvMjc2NTI0ODA2LWM2NzlhOWVlLTAyOTQtNDU3MS05ZmMwLWZkNzM2NGM4OThiOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMVQwNjM1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZWIwYWQyZjAyMDVhNjIwZDBmNjc0ZThmYTdhZWFjZGUxNjY0OTRkZTliMTEyMDdkOGRjNjU3MGZmNTI2ZTU0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6r1vNahj76bA1yXCcJzhs7yTSKQNVFL04R2t8kFJTB0)
> 
> Then I tried to install Karmada Dashboard by following the steps on github and I found out that I only had karmada-apiserver.config file (seems like I am missing something about karmada-host). Also, when I generated bearer token, it showed this error. ![image](https://private-user-images.githubusercontent.com/142280847/276524010-035bc0ac-94b3-4408-85f2-567a4af5cea5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMzkyMzcsIm5iZiI6MTcxMDEzODkzNywicGF0aCI6Ii8xNDIyODA4NDcvMjc2NTI0MDEwLTAzNWJjMGFjLTk0YjMtNDQwOC04NWYyLTU2N2E0YWY1Y2VhNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMVQwNjM1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTUwZTZkNmI0YjQwZTEwMGEwYTZjZTJjMWYxYWUxYTMxMzQ1YmI1OGUwODYyNzFhOGI5ZmU2YjVjODQxOTVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Q04A8shchhXVbLulFBZzQZaGwdKn-qQzZsdKDLpVx5Y)

TLDR:
if you want [temporary token](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-an-api-token-for-a-serviceaccount),  may be expired hours ago, you get it by executing follow shell script:
```shell
export KUBECONFIG=path/to/karmada-kubeconfig
kubectl -n karmada-system create token karmada-dashboard
```
if you want a [long lived token](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-long-lived-api-token-for-a-serviceaccount), you can following next steps
```shell
kubectl apply -f - <<EOF
apiVersion: v1
kind: Secret
metadata:
  name: karmada-dashboard-secret
  namespace: karmada-system
  annotations:
    kubernetes.io/service-account.name: karmada-dashboard
type: kubernetes.io/service-account-token
EOF

kubectl -n karmada-system get secret/karmada-dashboard-secret -ojsonpath='{.data.token}' | base64 -d && echo
```


It's not the problem of karmada. It's [KEP-1205](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1205-bound-service-account-tokens) which introduce the feature that caused not automatic generated token for serviceaccount. 
More information you can follow this link: https://karmada.io/docs/troubleshooting/trouble-shooting/#serviceaccount-deployed-to-the-karmada-control-plane-dont-generate-token
![image](https://github.com/karmada-io/dashboard/assets/6478745/11767972-18cb-4648-bfef-d76ac42bd06d)

**Comment by warjiang** [comment]
> The Karmada dashboard project is still far from being available, and the development of this project has actually stagnated due to the lack of front-end developers. We are also seeking developers who are willing to contribute to this project.

If there is any plan to maintain karma-dashboard, may be i can do it part time 🤣

**Comment by RainbowMango** [comment]
> If there is any plan to maintain karma-dashboard, may be i can do it part time

Glad to hear that and thank you! 
Can we talk about it at tomorrow's [community meeting](https://github.com/karmada-io/karmada?tab=readme-ov-file#meeting)?  Or just contact me by email.

**Comment by warjiang** [comment]
> > If there is any plan to maintain karma-dashboard, may be i can do it part time
> 
> Glad to hear that and thank you! Can we talk about it at tomorrow's [community meeting](https://github.com/karmada-io/karmada?tab=readme-ov-file#meeting)? Or just contact me by email.

I'll attend the meeting on time 😄

**Comment by warjiang** [comment]
The problem has been solved in the PR #93
